src/core/friendly_errors/validate_params.js

```diff
@@ -344,9 +344,9 @@ if (typeof IS_MINIFIED !== 'undefined') {
   const isNumber = param => {
     switch (typeof param) {
       case 'number':
-        return true;
+        return isFinite(param);
```

Comment:
I don't think finite number check should be implemented here as mentioned in the original issue there could still be utility for using `Infinity` in certain circumstances and if drawing functions are ever to enforce finite only numbers they should be done in the drawing functions themselves instead of in the general case here.